src/value.jl

```diff
@@ -80,48 +80,39 @@ Base.promote_rule(::Type{C1}, ::Type{C2}) where {C1<:CatValue, C2<:CatValue} =
 
 Base.convert(::Type{Ref}, x::CatValue) = RefValue{leveltype(x)}(x)
 Base.convert(::Type{String}, x::CatValue) = convert(String, get(x))
-Base.convert(::Type{Any}, x::CatValue) = x
 
 # Defined separately to avoid ambiguities
 Base.convert(::Type{AbstractString}, x::CategoricalString) = x
 Base.convert(::Type{T}, x::T) where {T <: CatValue} = x
 Base.convert(::Type{Union{T, Missing}}, x::T) where {T <: CatValue} = x
 Base.convert(::Type{Union{T, Nothing}}, x::T) where {T <: CatValue} = x
+
+@static if isdefined(Base, :unwrap)
```

Comment:
Please switch to a `VERSION` check once the PR is merged in Julia, so that Femtocleaner automatically removes the code the day we will require Julia 1.3.